awx/ui_next/src/screens/Job/WorkflowOutput/WorkflowOutputNode.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { Fragment } from 'react';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import styled from 'styled-components';
+import { StatusIcon } from '@components/Sparkline';
+import { WorkflowNodeTypeLetter } from '@components/Workflow';
+import { secondsToHHMMSS } from '@util/dates';
+import { JOB_TYPE_URL_SEGMENTS } from '@constants';
+import { constants as wfConstants } from '@util/workflow';
+
+const NodeG = styled.g`
+  cursor: ${props => (props.job ? 'pointer' : 'default')};
+`;
+
+const JobTopLine = styled.div`
+  display: flex;
+  align-items: center;
+  margin-top: 5px;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+
+  p {
+    margin-left: 10px;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+  }
+`;
+
+const Elapsed = styled.div`
+  text-align: center;
+  margin-top: 5px;
+
+  span {
+    font-size: 12px;
+    font-weight: bold;
+    background-color: #ededed;
+    padding: 3px 12px;
+    border-radius: 14px;
+  }
+`;
+
+const NodeContents = styled.foreignObject`
+  font-size: 13px;
+  padding: 0px 10px;
+`;
+
+const NodeDefaultLabel = styled.p`
+  margin-top: 20px;
+  text-align: center;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+`;
+
+function WorkflowOutputNode({
+  node,
+  nodePositions,
+  mouseEnter,
+  mouseLeave,
+  i18n,
+}) {
+  let borderColor = '#93969A';
+
+  if (node.job) {
+    if (
+      node.job.status === 'failed' ||
+      node.job.status === 'error' ||
+      node.job.status === 'canceled'
+    ) {
+      borderColor = '#d9534f';
+    }
+    if (node.job.status === 'successful' || node.job.status === 'ok') {
+      borderColor = '#5cb85c';
+    }
+  }
+
+  const handleNodeClick = () => {
+    if (node.job) {
+      window.open(
```

Comment:
The current behavior in awx is to redirect the current tab to the job instead of opening a new one. If that's what we want here, then we could probably use something from the router `history` api here instead of a raw window command.  Otherwise I think this would be fine.

Reply:
Should be addressed in https://github.com/ansible/awx/pull/5712